packages/core/content-manager/server/src/controllers/relations.ts

```diff
@@ -458,7 +458,7 @@ export default {
       ordering: 'desc',
     });
 
-    const relationsUnion = uniqBy('id', concat(sanitizedRes.results, res.results));
+    const relationsUnion = uniqBy('documentId', concat(sanitizedRes.results, res.results));
```

Comment:
Would it be possible to easily add a test (unit? API?) to make sure we're doing the unique transform on `documentId` and not `id`? Seems like a critical path given the issue this fix

Reply:
Yes I'll add some tests once the QA validates it fixes the issues as there are many :/